stdlib/ballerina-http/src/main/ballerina/ballerina/net/http/natives.bal

```diff
@@ -40,12 +58,15 @@ public native function <Connection conn> getSession () (Session);
 @Field {value:"method: HTTP request method"}
 @Field {value:"httpVersion: The version of HTTP"}
 @Field {value:"userAgent: User-Agent request header"}
+@Field {value:"extraPathInfo: Additional information associated with the URL sent by the client"}
+@Field {value:"protocol: The protocol of the HTTP request"}
```

Comment:
Shall we say "The protocol of the request"?  Because HTTP is also one of the protocols.